python/ray/tune/util.py

```diff
@@ -128,6 +130,18 @@ def _from_pinnable(obj):
     return obj[0]
 
 
+def recursive_fnmatch(dirpath, pattern):
```

Comment:
Yes, it was added by @richardliaw for search the `checkpoint-1` without knowing all intermediate directories, in the previous `test_tune_restore.py` which only restore an absolute path checkpoint. 

But now since we indeed need to know intermediate directories for assertions, so this function is not used.

Reply:
I see, this seems to be related to another PR. Should be fine after rebase/merge.